catalog/views.py

```diff
@@ -1,8 +1,18 @@
 from django.shortcuts import render
 
+from .models import Product
 
-def home(request):
-    return render(request, "catalog/home.html")
+
+def products_list(request):
+    products = Product.objects.all()
+    context = {"products": products}
+    return render(request, "catalog/products_list.html", context)
+
+
+def product_detail(request, pk):
+    product = Product.objects.get(pk=pk)
```

Comment:
Если передать несуществующий id продукта, то упадет 500 ошибка, что не хорошо

Правильнее возвращать 404 (NotFound), что такого продукта нет

Поэтому получать продукт нужно так:

```suggestion
    product = get_object_or_404(Product, pk=pk)
```